### ISSUE TITLE
issue while running on server

### ISSUE THREAD

**Issue by GeekySudhir** [animeshsinghweb/simple-chat#1]
i am having following issues while running the fork repo. on my server any help?

![deepinscreenshot_20171008120123](https://user-images.githubusercontent.com/28150975/31314622-7a3f3ffe-ac22-11e7-9d31-04b5eadfe6f6.png)

**Comment by ashishsah1000** [comment]
Its working properly you must have changed something...did you run the command npm install ?

**Comment by animeshsinghweb** [comment]
Yeah, @SuperdudeSudhir you need to run `npm install` while navigating inside the repo.
